packages/web-twig/tests/__snapshots__/ComponentsSnapshotTest__test with data set textFieldBaseMultiline.twig__1.html

```diff
@@ -0,0 +1,10 @@
+<html><body>
+<div class="Textarea   Textarea--error">
+    <label for="example" class="Textarea__label  Textarea__label--required">
```

Comment:
Please, rebase your code. The problem with multiple spaces between classes should be solved by https://github.com/lmc-eu/spirit-design-system/pull/483.